samples/pollDiscoveryOccurrenceFinished.js

```diff
@@ -0,0 +1,54 @@
+// [START containeranalysis_poll_discovery_occurrence_finished]
+// Repeatedly query the Container Analysis API for the latest discovery occurrence until it is
+// either in a terminal state, or the timeout value has been exceeded
+const pollDiscoveryOccurrenceFinished = async (
```

Comment:
It would probably be helpful to add more inline comments to this function to make it easier to follow